database/ffldb/blockio.go

```diff
@@ -402,6 +403,15 @@ func (s *blockStore) writeData(data []byte, fieldName string) error {
 	n, err := wc.curFile.file.WriteAt(data, int64(wc.curOffset))
 	wc.curOffset += uint32(n)
 	if err != nil {
+		if pathErr, ok := err.(*os.PathError); ok {
+			if errno, ok := pathErr.Err.(syscall.Errno); ok &&
+				errno == syscall.ENOSPC {
+
+				log.Errorf("%v. Cannot save any more blocks "+
```

Comment:
can we also mention it's due to the disk being full so the operator can act accordingly?

Reply:
Addressed in the latest commit